src/uwtools/scheduler.py

```diff
@@ -181,20 +181,23 @@ def _managed_directives(self) -> dict[str, Any]:
     @property
     def _prefix(self) -> str:
         """
-        Returns the scheduler's resource-request prefix.
+        The scheduler's resource-request prefix.
         """
         return "#BSUB"
 
     @property
     def _processed_props(self) -> dict[str, Any]:
+        """
+        Pre-processed runscript directives,
+        """
```

Comment:
I'm curious how this one passed the linter...

Reply:
https://github.com/pylint-dev/pylint/issues/3177#issuecomment-539981952 -- which was news to me, but testing it out it seems to be true.